### ISSUE TITLE
No such file or directory error on OS X due to Dos line endings instead Unix in npm package

### ISSUE THREAD

**Issue by dhax** [jeffijoe/icebug#1]
```
npm run debug

> koa-es7-boilerplate@0.0.7 debug /Users/node/github/koa-es7-boilerplate
> icebug src/bin/server.js

env: node\r: No such file or directory

npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/barnaba/.nvm/versions/node/v5.6.0/bin/node" "/Users/barnaba/.nvm/versions/node/v5.6.0/bin/npm" "run" "debug"
npm ERR! node v5.6.0
npm ERR! npm  v3.6.0
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! koa-es7-boilerplate@0.0.7 debug: `icebug src/bin/server.js`
npm ERR! spawn ENOENT
```

As a workaround one can just locally convert node_modules/icebug/bin/icebug.js to LF format, although only the fist line here is causing the error

**Comment by jeffijoe** [comment]
Thanks! I'll look into this once I get a chance to try it out on a Mac.

If you feel like it, you can submit a PR with a fix. :)

**Comment by dhax** [comment]
The problem might be by npm publishing from windows with misconfigured git client, i.e. not replacing CRLF with LF

**Comment by jeffijoe** [comment]
@DiknT my git client is configured with auto-crlf=true, so it should be all set?

**Comment by ddproxy** [comment]
https://github.com/npm/npm/issues/4607

It's not your fault, directly.

**Comment by ddproxy** [comment]
@DiknT What version node and npm are you running?

 npm -v
3.6.0
 node -v
v5.6.0

And I'm having the same issue. I'm investigating a fix but short term you can modify the node_modules/icebug/bin/icebug.js file and change it's formatting to LF instead of CRLF

**Comment by dhax** [comment]
latest as shown in above error log.
maybe having a .gitattributes file on a windows machine fixes this when npm publishing. From the linked issues on this topic it doesn't look like npm is about to do anything about it

**Comment by jeffijoe** [comment]
@ddproxy thanks for the PR, @DiknT could you try it out and see if it works? Will close this if it fixes it for you.

**Comment by dhax** [comment]
looks good! Thanks

**Comment by ddproxy** [comment]
Fixed issue in Koa Boilerplate - good to go

**Comment by jeffijoe** [comment]
Sweet - was published as 0.1.4 on npm. Closing, thanks all. :)
